### PR TITLE
Backup popup

### PR DESCRIPTION
**Pull request by jorgeblacio**
- Now there is a reminder for users to activate auto-backup.
- Auto backup returned to Job since instancing a Foreground Service within a Job was causing trouble with backups.

### CODE REVIEW

File: src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -261,7 +261,7 @@ schwarze Liste gesetzt wird</string>
 
         Wollen Sie fortfahren?]]></string>
   <string name="sign_in_warning_dialog_title">Anmeldungswarnung</string>
-  <string name="password_warning_dialog_title">Warnung</string>
+  <string name="password_warning_dialog_title">Warnung ⚠️</string>
```
Comment (danieltigse) [inline src/main/res/values-de/strings.xml:264]:
```suggestion
  <string name="password_warning_dialog_title">Warning ⚠️</string>
```

File: src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -740,4 +740,11 @@ Möchten Sie sie verwerfen und fortfahren?</string>
   <string name="block_remote_dialog_title">Turn Off Image Block</string>
   <string name="block_remote_dialog_message">Your security is affected by deactivating this function. \n\n You can reactivate this function by going to Settings->Privacy.</string>
   <string name="download_web_view_started">Your download for has started.</string>
+  <string name="recommend_backup_title">Auto Backup to Drive</string>
+  <string name="recommend_backup_message">Activate backups to guarantee you never loose your data.</string>
+  <string name="recommend_backup_ok">Turn on Auto Backup</string>
+  <string name="recommend_backup_not_now"><u>Not now</u></string>
+  <string name="recommend_backup_warning_message">Your emails are exclusively kept on your device, not on Criptext servers. \nThis means that if you delete he app or loose the device you will loose all your emails.</string>
```
Comment (danieltigse) [inline src/main/res/values-de/strings.xml:747]:
```suggestion
  <string name="recommend_backup_warning_message">Your emails are exclusively kept on your device, not on Criptext servers. \nThis means that if you delete the app or loose the device you will loose all your emails.</string>
```

File: src/main/res/values-es/strings.xml [hunk 3]
```diff
@@ -712,4 +712,11 @@
   <string name="block_remote_dialog_title">Apagar Bloqueo de Imágenes</string>
   <string name="block_remote_dialog_message">Tu seguridad puede verse comprometida al desactivar este bloqueo. \n\nPuedes volver a reactivarlo desde Configuración->Privacidad.</string>
   <string name="download_web_view_started">Your download for has started.</string>
+  <string name="recommend_backup_title">Auto Backup to Drive</string>
```
Comment (danieltigse) [inline src/main/res/values-es/strings.xml:715]: translate it

File: src/main/kotlin/com/criptext/mail/scenes/mailbox/MailboxSceneController.kt [hunk 4]
```diff
@@ -783,6 +825,13 @@ class MailboxSceneController(private val scene: MailboxScene,
         scene.showDialogLabelsChooser(LabelDataHandler(this))
     }
 
+    private fun checkRecommendBackup(threadSize: Int){
+        if(threadSize > 10
+                && !storage.getBool(KeyValueStorage.StringKey.HasBeenAskedRecommendBackup, false)){
```
Comment (danieltigse) [inline src/main/kotlin/com/criptext/mail/scenes/mailbox/MailboxSceneController.kt:830]: This doesn't work for me. The popup was shown even thought I had backup enabled.

Comment (danieltigse) [reply src/main/kotlin/com/criptext/mail/scenes/mailbox/MailboxSceneController.kt:830]: Also, the validation is for each account

File: src/main/res/values-de/strings.xml (hunk 1 above)
Comment (jorgeblacio) [reply src/main/res/values-de/strings.xml:264]: This is the German version, it was already translated so why not leave it like that?

File: src/main/res/values-es/strings.xml [hunk 5]
```diff
@@ -711,5 +711,12 @@
   <string name="block_remote_turn_off_message_disable">Contenido remoto deshabilitado para la cuenta: %1$s</string>
   <string name="block_remote_dialog_title">Apagar Bloqueo de Imágenes</string>
   <string name="block_remote_dialog_message">Tu seguridad puede verse comprometida al desactivar este bloqueo. \n\nPuedes volver a reactivarlo desde Configuración->Privacidad.</string>
-  <string name="download_web_view_started">Your download for has started.</string>
+  <string name="download_web_view_started">Tu descarga ha comenzado.</string>
+  <string name="recommend_backup_title">Respaldo Automático a Drive</string>
+  <string name="recommend_backup_message">Cuando activas el respaldo automático te aseguras de no perder tu información.</string>
+  <string name="recommend_backup_ok">Activar Respaldo Automático</string>
+  <string name="recommend_backup_not_now"><u>No por ahora</u></string>
+  <string name="recommend_backup_warning_message">Tus correos solo existen en tu dispositivo, no en los servidores de Criptext. \nEsto quiere decir que si desinstalas tu app o pierdes tu dispositivo, perderas también todos tus correos.</string>
```
Comment (danieltigse) [inline src/main/res/values-es/strings.xml:719]:
```suggestion
  <string name="recommend_backup_warning_message">Tus correos solo existen en tu dispositivo, no en los servidores de Criptext. \nEsto quiere decir que si desinstalas tu app o pierdes tu dispositivo, perderás también todos tus correos.</string>
```

**Pull request merged**